Dockerfile

```diff
@@ -36,4 +36,4 @@ ARG UI_TGZ=""
 
 ENV INVENIO_COLLECT_STORAGE='flask_collect.storage.file'
 
-RUN ./scripts/bootstrap --deploy ${UI_TGZ} && rm -fr ui/node_modules
+RUN pipenv run bootstrap --deploy ${UI_TGZ} && rm -fr ui/node_modules
```

Comment:
I think we can delete `rm -fr ui/node_modules` as we don't have `ui` anymore. Do you confirm that?